index.html

```diff
@@ -12,34 +12,36 @@
     </head>
     <body>
       <div id="ui">
-	<div id="header">
-	  <div id="title"><h3>Pitt Open Data Project</h3></div>
-	  <div id="navigation">
-	    <ul>
-	      <li><a href="index.html">Home</a></li>
-	      <li><a href="about.html">About</a></li>
-	    </ul>
-	  </div>
-	</div>
-	<div id="sidebar">
-	  <div id="controls">
-	    <h3>Date Range</h3>
-	    <form name="dateSelection">
-	      <div class="dateRadio">
-		<input type="radio" name="radioDate" id="radioMonth" checked>
-		<label for="radioMonth">Past 30 Days</label>
-	      </div>
-	      <div class="dateRadio">
-		<input type="radio" name="radioDate" id="radioWeek">
-		<label for="radioWeek">Past 7 Days</label>
-	      </div>
-	      <div class = "dateRadio">
-		<input type="radio" name="radioDate" id="radioDay">
-		<label for="radioDay">Today</label>
-	      </div>
-	    </form>
-	  </div>
-	</div>
+      	<div id="header">
+      	  <div id="title"><h3>Pitt Open Data Project</h3></div>
+      	  <div id="navigation">
+      	    <ul>
+      	      <li><a href="index.html">Home</a></li>
+      	      <li><a href="about.html">About</a></li>
+      	    </ul>
+      	  </div>
+      	</div>
+        <div id="mobileMenuIcon"></div>
```

Comment:
It turns out we want to expand/contract on desktop, too, so we want to show this either way, and we should avoid the `mobile` terminology/design.

Instead of a hamburger icon, maybe we could have a small right-pointing arrow that expands onclick and turns into a left-pointing arrow in the top right corner of the UI that contracts onclick.

Also, it seems broken on some mobile browsers, and at least for me the `hide` is never shown on the UI after the icon is clicked